config/toml_config.go

```diff
@@ -88,6 +89,15 @@ type ExtraConfig struct {
 	EnableSLBLoadBalance bool
 }
 
+type LogConfig struct {
+	LogFile            string
+	SingleFileMaxValue int
+	MaxBackups         int
+	MaxAge             int
```

Comment:
I think you need to add some code comment to declare the function of each variable